.env

```
@@ -1 +1,18 @@
 
+# Database
+DATABASE_URL="postgresql://neondb_owner:npg_o2IJlsLu0VWU@ep-empty-butterfly-ah75718y-pooler.c-3.us-east-1.aws.neon.tech/neondb?sslmode=require&channel_binding=require"
+
+# Auth
+BETTER_AUTH_SECRET=TupbhIhqzRzucqKbrTvrV0eMS9aqIN69
+BETTER_AUTH_URL=http://localhost:3000 # Base URL of your app
```

dotenv-linter (4.0.0)

[warning] [ValueWithoutQuotes] This value needs to be surrounded in quotes

(ValueWithoutQuotes)